charts/kong/crds/custom-resource-definitions.yaml

```diff
@@ -1,425 +1,991 @@
+# generated using: kubectl kustomize github.com/kong/kubernetes-ingress-controller/railgun/config/crd?ref=main
```

Comment:
This is a big fat breaking change that must have a huge warning in the release notes. We're effectively dropping support for k8s versions too old for v1.

Reply:
I discussed this with @rainest before doing it, and yes it is breaking (for old kubernetes clusters). Would be helpful if you could bullet point list the specific actions you want to see taken to consider this comment resolved please :bow: